docker/jukebox.Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN usermod -aG pulse ${USER}
 # Install all Jukebox dependencies
 RUN apt-get update && apt-get install -qq -y \
     --allow-downgrades --allow-remove-essential --allow-change-held-packages \
-    gcc at wget \
+    gcc g++ at wget \
```

Comment:
This was never a problem for me. What's the error message?

Reply:
The error message was that the g++ compiler could not be found. 

I built the container under Windows, but it should not make a difference which host. 

I'm wondering, if zmq changed something which requires now the g++ instead of gcc only.

I got other errors in the webapp after fixing that, but it was too late last night to investigate.

Reply:
Error was:
```
190.9   error: command 'x86_64-linux-gnu-g++' failed: No such file or directory
190.9   ERROR: Failed building wheel for pyzmq
190.9 ERROR: Could not build wheels for pyzmq which use PEP 517 and cannot be installed directly
------
failed to solve: process "/bin/sh -c pip install -v --no-binary pyzmq --pre pyzmq" did not complete successfully: exit code: 1
```

Hat bei mir geholfen!!